how-to-git.md

```diff
@@ -0,0 +1,43 @@
+# How to Use git When Contributing to Our Projects
+
+The goal is to fork and preserve the state of the project at the time that you forked it, while contributing development branches to the original/upstream project.
+
+Keeping your forked/master in tact allows others to more easily work with your forked copy.
+
+Here are the steps with links to documentation:
+
+1. [Fork to your GitHub account](https://help.github.com/articles/fork-a-repo/#fork-an-example-repository)
+1. [Clone your forked copy onto your computer](https://help.github.com/articles/fork-a-repo/#step-2-create-a-local-clone-of-your-fork)
+1. [Set master project repo as the upstream for your cloned repo](https://help.github.com/articles/configuring-a-remote-for-a-fork/)
+1. Create a new branch **(don't develop on 'Master')**
+    ```
```

Comment:
Sorry I omitted this before, but you can put the syntax name (in this case "bash") after the ``` for highlighting, compare:

```
$ echo "without the bash name behind the \`\`\`"
```

```bash
$ echo "with the bash name behind the \`\`\`"
```